### PR TITLE
Code Quality: Fixed an issue where backdrop was not active when resuming from background

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Steps used to test these changes

1. Open Files app
2. Close
3. Resume from the Taskbar icon
4. See SystemBackdrop is working normally without any further interactions

### CODE REVIEW

File: src/Files.App/App.xaml.cs [hunk 1]
```diff
@@ -159,6 +159,8 @@ async Task ActivateAsync()
 				}
 
 				await AppLifecycleHelper.InitializeAppComponentsAsync();
+
+				AppLifecycleHelper.IsLaunchInitialized = true;
```
Comment (yaira2) [inline src/Files.App/App.xaml.cs:163]: Can you add a comment?

File: src/Files.App/Helpers/Application/AppLifecycleHelper.cs [hunk 2]
```diff
@@ -58,6 +58,12 @@ public static class AppLifecycleHelper
 				_ => Constants.AssetPaths.StableLogo
 			});
 
+		/// <summary>
+		/// Gets or sets a value that indicates whether the application is ready to be interacted.
```
Comment (yaira2) [inline src/Files.App/Helpers/Application/AppLifecycleHelper.cs:62]:
```suggestion
		/// Gets or sets a value that indicates whether the application is ready to be interacted with.
```

File: src/Files.App/Helpers/UI/AppSystemBackdrop.cs
Comment (yaira2) [inline src/Files.App/Helpers/UI/AppSystemBackdrop.cs:1]: What was changed here?

Comment (0x5bfa) [reply src/Files.App/Helpers/UI/AppSystemBackdrop.cs:1]: I had to expose SystemBackdropConfiguration

Comment (0x5bfa) [reply src/Files.App/Helpers/UI/AppSystemBackdrop.cs:1]: @heftymouse fyi.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**